packages/e2e/features/ui/components/authenticator/sign-in-sms-mfa.feature

```diff
@@ -12,22 +12,15 @@ Feature: Sign In with SMS MFA
     Then I type my "phone number" with status "CONFIRMED"
     Then I type my password
     Then I click the "Sign in" button
+    Then I will be redirected to the confirm sms mfa page
     Then I see "Confirm SMS Code"
     Then I type a valid SMS confirmation code
     Then I spy request '{ "headers": { "X-Amz-Target": "AWSCognitoIdentityProviderService.RespondToAuthChallenge" } }'
     Then I click the "Confirm" button
     Then I confirm request '{"headers": { "X-Amz-Target": "AWSCognitoIdentityProviderService.RespondToAuthChallenge" } }'
 
   @angular @react @vue
-  Scenario: Sign in using a valid phone number and SMS MFA
-    When I select my country code with status "CONFIRMED"
-    Then I type my "phone number" with status "CONFIRMED"
-    Then I type my password
-    Then I click the "Sign in" button
-    Then I will be redirected to the confirm sms mfa page
```

Comment:
Only difference between this test and the above was checking for the confirm sign in page, added the check to to line 15